index.js

```diff
@@ -179,9 +184,12 @@ Color.prototype = {
 		return rgb;
 	},
 
-	round(places) {
-		places = Math.max(places || 0, 0);
-		return new Color([...this.color.map(roundToPlace(places)), this.valpha], this.model);
+	round() {
+		if ((arguments?.length ?? 0) !== 0) {
+			console.warn('Color.round() no longer accepts a precision argument and now rounds to the nearest integer. Consider using Color.toFixed() if you want to retrieve color elements with a precision argument.');
```

Comment:
Don't log anything to users since this is a CLI widely used package. Your log may break some apps.

Reply:
I agree with you, but I need some sort of safeguard here.

Calling with an unused argument (the old way) will not cause any error _per se_ but will cause the library to return colours that are rounded differently. It's a silent data corruption that users would need to spend time finding the root cause of.

There is a breaking change notice describing this API change, but of course many people upgrade breaking versions without reading changelogs. I could explicitly throw, which would at least more predictably solve the underlying issue I want to solve. But doing and logging nothing would likely be detrimental here.

Reply:
This can be released as a new major since it is breaking. Let's just `throw new TypeError('.round() takes no arguments')` here.

Also you can just do `if (arguments?.length)` looking at this again.